specs/root.md

```diff
@@ -42,6 +42,7 @@ Specifications of new features in active development.
   - [Fault Dispute Game](./experimental/fault-proof/fault-dispute-game.md)
     - [Honest Challenger Behavior](./experimental/fault-proof/honest-challenger-fdg.md)
   - [Cannon VM](./experimental/fault-proof/cannon-fault-proof-vm.md)
+- [Interop](./interop.md)
```

Comment:
This will need an update based on the latest commits that move the single file into a directory